core-java-8/pom.xml

```diff
@@ -94,6 +94,11 @@
             <artifactId>streamex</artifactId>
             <version>${streamex.version}</version>
         </dependency>
+        <dependency>
+			<groupId>joda-time</groupId>

```

Comment:
The whitespacing is still incorrect here. Should be spaces, like the rest of the file.

Reply:
Done 👍